en/docs/get-started/try-ciam-cloud.md

```diff
@@ -0,0 +1,19 @@
+# Try CIAM Cloud
+
+CIAM Cloud provides you the opportunity to try out CIAM Cloud on a sandboxed environment. This documentation will guide you on how to set up you CIAM Cloud Sandbox environment.
+
+To request access to try out Private CIAM Cloud Sandbox environment:
+
+1. Go to [WSO2 Private CIAM Cloud](https://wso2.com/ciam-suite/private-ciam-cloud/b2b-ciam/) page, and click on **Try Now**.
+2. Enter your details on the form that appears.
+
+    <img src="../../../assets/img/intro/try-out-ciam-sandbox.png" alt="Try out Private CIAM Cloud" width="700" style="border:1px solid grey">
+
+3. Click **Submit** to send your details.
+
```

Comment:
Once the user submits the "Try on Private CIAM Cloud" form, WSO2 will contact the user for more information. After more information is shared, an account will be created.
 
Let's document that